shared-ts/enums.ts

```diff
@@ -176,3 +176,8 @@ export enum SocialMediaPostStatus {
     successful = "Successful",
     rejected = "Rejected",
 }
+
```

Comment:
sometimes when i change a mode of transport the services / stops clear - is this expected?